scripts/test-sig-algs.py

```diff
@@ -487,6 +460,16 @@ def main():
             bad += 1
             failed.append(c_name)
 
+    print("Check if server correctly selects signature algorithm for SKE")
+    print("Test to verify that Server Key Exchange is signed with safe")
+    print("and correct algorithms.")
+    print("Note that test expects server with support for both rsa_pss_rsae_*")
+    print("and rsa_pss_pss_* signatures, in other words, one with both")
+    print("rsaEncryption key in one certificate and rsasse-pss in second")
+    print("certificate. If there's only one certificate installed in server,")
+    print("some of the tests that advertise just one algorithm may need to be")
+    print("disabled.")
+
```

Comment:
So when testing NSS for instance. Do we need two server certificates in NSS DB? Or should we execute rsa_pss_pss_* and  rsa_pss_rsae_* probes separately in two executions? Perhaps I am missing something truly obvious here but I must admit that I am a bit confused here.

Reply:
either option will work

the note is here as tlslite-ng doesn't support multiple certificates, but other implementations do

but even if the implementation does support multiple certificates set up at the same time, doing two runs is a valid test (at the same time, for full coverage, it does mean the test needs to be run three times)

Reply:
Thanks. I still sense some fragility in the tests. I tried approach you suggested and did three executions:

1. Server uses NSS DB "for" rsa_pss_pss certificates - all tests but rsa_pss_rsae_* passed.
2. Server uses NSS DB "for" rsa_pss_rsae certificates - all but the following tests passed:  'empty sigalgs', 'extra sigalgs', 'MD5 first', 'MD5 first, no SHA-1', 'no SHA-1' and 'rsa_pss_pss_* only' ".
3. Server uses NSS DB with both certs - all test but the following passed: 'empty sigalgs', 'extra sigalgs', 'MD5 first', 'MD5 first, no SHA-1', 'no SHA-1'.

Reply:
All issues resolved. Thanks!